docs/dev/README.md

```diff
@@ -1,35 +1,52 @@
 ---
-title: Introduction to Agent-based Integrations
+title: Build an Integration
 kind: documentation
+description: Learn how to develop and publish an offering on the Integrations page.
 aliases:
-  - /guides/agent_checks/
-  - /agent/agent_checks
-  - /developers/agent_checks/
+- /guides/agent_checks/
+- /agent/agent_checks
+- /developers/agent_checks/
+dependencies: "https://github.com/DataDog/integrations-core/blob/alai97/add-marketplace-documentation/docs/dev/README.md"
+further_reading:
+- link: "/developers/"
+  tag: "Documentation"
+  text: "Learn how to develop on the Datadog platform"
 ---
 
-## Why create an integration?
+## Overview
+
+This page walks through how Technology Partners can build an out-of-the-box integration using the Datadog Agent or Datadog API. 
+
+The [Integrations page][3] includes integrations and custom dashboard widgets built by both Datadog and our Technology Partners, available at no cost to Datadog customers. 
 
-[Custom Checks][1] are great for occasional reporting, or in cases where the data source is either unique or very limited. For more general use-cases - such as application frameworks, open source projects, or commonly-used software - it makes sense to write an integration.
+The [Datadog Marketplace][4], on the other hand, is a commercial platform for Technology Partners to _sell_ a variety of offerings, including integrations, custom dashboard widgets, software subscriptions/licenses, and professional services to Datadog customers.
 
-Metrics reported from accepted integrations are not counted as custom metrics, and therefore don't impact your custom metric allocation. (Integrations that emit potentially unlimited metrics may still be considered custom.) Ensuring native support for Datadog reduces friction to adoption, and incentivizes people to use your product, service, or project. Also, being featured within the Datadog ecosystem is a great avenue for added visibility.
+## Datadog Integrations
 
-### What's the process?
+### [API-based Integration][1]
+API-based integrations can enrich a customer's Datadog environment by submitting data from external technologies via the Datadog API. Customers authorize API integrations access to their accounts through OAuth, and partners write and host the implementation code that makes up the integration. API integrations work well for partners building a connector between Datadog and another SaaS platform.
 
-The initial goal is to generate some code that collects the desired metrics in a reliable way, and to ensure that the general integration framework is in place. Start by writing the basic functionality as a custom Check, then fill in the framework details from [Create a New Integration][2].
+### [Agent-based Integration][2]
```

Comment:
Can we switch the ordering and have the "Agent-based Integration" section go before "API-based Integration," since in the initial paragraph we mention Agent Integrations first?